test/1.test-common.js

```diff
@@ -42,6 +42,8 @@ const debugOptions = {
   enable: false,
 }
 
+
+// TODO BAM update these settings for ace-context.
```

Comment:
Will this `TODO` be addressed?

Reply:
it will be. i created a jira task to go through all my outstanding todos. because it's for debugging context problems in test suite only i wanted to defer it as it requires changes to ace-context.